ui/raidboss/data/05-shb/trial/diamond_weapon-ex.txt

```diff
@@ -0,0 +1,116 @@
+### Diamond Weapon Ex
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 10000 jump 0
+
+# -ii 5FEB 5FBE 5FAA 5FEC 5FBA 5FFD 5FBB 6055 6159 5FF9 5FCD 5FFA 5FF9 612E 5FB3 5FCE 5FD2 5FFE 5FD3 5FB4 5FB6 5FB1 5FB0 5FBF 5FAE 5FA6 5FC3 5FC9 5FCA 5FC4
+
+### Phase 1
+# There are no autoattacks from the boss until after the initial raidwide, so nothing to sync against for autos
+0 "Start" sync /Engage!/ window 1,1
+21.3 "Diamond Rain" sync /:The Diamond Weapon:5FA7:/ window 25,10
```

Comment:
You could put a `sync /:5FA7:The Diamond Weapon/` before this to start syncing earlier too in case somebody pulls without a countdown (or there's an egregious early pull).

Reply:
I've added what I think you meant, but not 100% sure. Can't get in-game to test it and haven't tracked down the raidemulator sync issue for the timeline yet.

Reply:
`test_timeline.py` is usually the tool I go to to make sure timelines look good across a variety of logs.  raidemulator is good for making sure triggers all work correctly, but `test_timeline.py` helps with finding a lot of timeline-specific issues.